images/prometheus/tests/main.tf

```diff
@@ -148,17 +148,28 @@ resource "helm_release" "kube-prometheus-stack" {
   }
 }
 
-data "oci_exec_test" "check-prometheus" {
-  digest      = var.digests["core"]
-  script      = "./check-kube-prometheus-stack.sh"
-  working_dir = path.module
-  depends_on  = [helm_release.kube-prometheus-stack]
-}
-
 data "oci_exec_test" "node-runs" {
-  depends_on = [data.oci_exec_test.check-prometheus]
+  depends_on = [resource.helm_release.kube-prometheus-stack]
 
   digest      = var.digests["node-exporter"]
   script      = "./node-runs.sh"
   working_dir = path.module
 }
+
+resource "helm_release" "cloudwatch-exporter" {
+  name       = "cloudwatch-exporter"
+  repository = "https://prometheus-community.github.io/helm-charts"
+  chart      = "prometheus-cloudwatch-exporter"
+
+  namespace        = "prometheus-cloudwatch-exporter"
+  create_namespace = true
+
+  set {
+    name  = "image.repository"
+    value = data.oci_string.ref["cloudwatch-exporter"].registry_repo
+  }
+  set {
+    name  = "image.tag"
+    value = "latest"
```

Comment:
The image won't be tagged yet, since it hasn't passed tests. I think you want `oci_string.ref["..."].pseudo_tag`

Reply:
Just fixed this!